lib/views/pay/qr_screen.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:clnapp/model/app_model/generate_invoice.dart';
+import 'package:clnapp/utils/app_provider.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:qr_flutter/qr_flutter.dart';
+
+class QrScreen extends StatefulWidget {
+  final AppProvider provider;
+  final AppInvoice invoice;
+
+  const QrScreen({Key? key, required this.invoice, required this.provider})
+      : super(key: key);
+
+  @override
+  State<QrScreen> createState() => _QrScreenState();
+}
+
+class _QrScreenState extends State<QrScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Generated QR'),
+      ),
```

Comment:
```suggestion
      appBar: AppBar(
        title: const Text('Generated QR'),
      ),
```

Is it possible to remove the title?